dump_zenodo_open_metadata.py

```diff
@@ -60,11 +60,11 @@ def parse_record(rec, spam=False):
 }
 
 choices = [k + ' - ' + v for k, v in DUMP_TYPES.items()]
-choice = str(input("What type of dump you want? {}".format(', '.join(choices)) + '\n'))
+choice = str(raw_input("What type of dump you want? {}".format(', '.join(choices)) + '\n'))
 dump_type = DUMP_TYPES.get(choice)
 
 if not dump_type:
-    raise Exception('Invalid dump type {}'.format(choice))
+    dump_type = 'full'
```

Comment:
```python
dump_type = DUMP_TYPES.get(choice, 'full')
```

And then we can remove the `if ...` :)

Reply:
This is a very nice improvement! Thank you!